docs/take-streaming-backup.md

```diff
@@ -59,13 +59,13 @@ were both compressed and encrypted, they must be decrypted before they are uncom
 |---------|------|
 | Stream the backup into an archived named `backup.xbstream` | `$ xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream`|
```

Comment:
Another suggestion is to remove the $ in all the command line strings. Its easy to copy this command line string. One less work to remove the $ sign